include/aie/Dialect/AIE/IR/AIE.td

```diff
@@ -1363,50 +1358,50 @@ def AIE_ShimDMAAllocationOp : AIE_Op<"shimDMAAllocation", [HasParent<"DeviceOp">
   );
   let results = (outs);
   let assemblyFormat = [{
-    $sym_name `(` $channelDir `,` $channelIndex `,` $col `)` attr-dict  
+    $sym_name `(` $channelDir `,` $channelIndex `,` $col `)` attr-dict
   }];
 }
 
 def AIE_ObjectFifoCreateOp: AIE_Op<"objectFifo", [HasParent<"DeviceOp">, Symbol]> {
   let summary = "Create a circular buffer or channel between two tiles";
   let description = [{
-    The `aie.objectFifo` operation creates a circular buffer established between a producer and one or 
-    more consumers, which are `aie.tile` operations. The`aie.objectFifo` instantiates the given number of 
-    buffers (of given output type) and their locks in the Memory Module of the appropriate tile(s) after 
-    lowering, based on tile-adjacency. These elements represent the conceptual depth of the `objectFifo` or, 
```

Comment:
I don't know how these are being erased - I tried to put them back but was unsuccessful.